src/ralph/api/utils.py

```diff
@@ -69,7 +73,9 @@ def get_serializer(self, *args, **kwargs):
                     args[0].__class__
                 ]
             except KeyError:
-                pass
```

Comment:
On `KeyError` this `serializer_class = self.get_serializer_class()` will be used.
Is this ok?